deepchecks/vision/checks/data_integrity/property_label_correlation.py

```diff
@@ -0,0 +1,190 @@
+# ----------------------------------------------------------------------------
+#
+"""Module contains the property label correlation check."""
+
+from collections import defaultdict
+from typing import Any, Dict, Hashable, List, Optional, TypeVar
+
+import pandas as pd
+
+import deepchecks.ppscore as pps
+from deepchecks import CheckResult, ConditionCategory, ConditionResult
+from deepchecks.core import DatasetKind
+from deepchecks.core.check_utils.feature_label_correlation_utils import get_pps_figure, pd_series_to_trace
+from deepchecks.core.errors import ModelValidationError
+from deepchecks.utils.strings import format_number
+from deepchecks.vision import Batch, Context, SingleDatasetCheck
+from deepchecks.vision.task_type import TaskType
+from deepchecks.vision.utils.image_properties import default_image_properties
+from deepchecks.vision.utils.property_label_correlation_utils import (calc_properties_for_property_label_correlation,
+                                                                      is_float_column)
+
+__all__ = ['PropertyLabelCorrelation']
+
+pps_url = 'https://docs.deepchecks.com/en/stable/checks_gallery/vision/' \
+          'train_test_validation/plot_feature_label_correlation_change.html'
+pps_html = f'<a href={pps_url} target="_blank">Predictive Power Score</a>'
+
+FLC = TypeVar('FLC', bound='PropertyLabelCorrelation')
+
+
+class PropertyLabelCorrelation(SingleDatasetCheck):
+    """
+    Return the Predictive Power Score of image properties, in order to estimate their ability to predict the label.
+
+    The PPS represents the ability of a feature to single-handedly predict another feature or label.
+    In this check, we specifically use it to assess the ability to predict the label by an image property (e.g.
+    brightness, contrast etc.)
+    A high PPS (close to 1) can mean that there's a bias in the dataset, as a single property can predict the label
+    successfully, using simple classic ML algorithms - meaning that a deep learning algorithm may accidentally learn
+    these properties instead of more accurate complex abstractions.
+    For example, in a classification dataset of wolves and dogs photographs, if only wolves are photographed in the
+    snow, the brightness of the image may be used to predict the label "wolf" easily. In this case, a model might not
+    learn to discern wolf from dog by the animal's characteristics, but by using the background color.
+
+    For classification tasks, this check uses PPS to predict the class by image properties.
+    For object detection tasks, this check uses PPS to predict the class of each bounding box, by the image properties
+    of that specific bounding box.
+
+    Uses the ppscore package - for more info, see https://github.com/8080labs/ppscore
+
+    Parameters
+    ----------
+    image_properties : List[Dict[str, Any]], default: None
+        List of properties. Replaces the default deepchecks properties.
```

Comment:
Are we also accepting static properties? If so would mention it + test

Reply:
@Nadav-Barak the static properties are passed to the run, not the init. They're managed by the context and already tested there, do you see a particular reason to test it separately?